Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -220,6 +220,23 @@ function xASL_wrp_ResampleASL(x)
         % Apparently, subtraction was already done on the scanner/reconstruction
         fprintf('%s\n', PathPWI{iSpace});
         xASL_io_SaveNifti(PathASL{iSpace}, PathPWI{iSpace}, ASL_im, 32, false);
+	elseif x.modules.asl.bDeltaM
+		% We have subtraction images 
+		
+		% For multi-PLD, save both PWI4D and PWI

```

Comment:
Also for single PLD, we should save both! This should have been there already, why did you change this?

Reply:
OK. Will do that. But it wasn't really me who changed that.

Reply:
I've checked again the code and currently, it always save PWI and PWI4D with the exception of single-volume or single-PLD files, where it saves PWI.nii only. 

But BASIL, later on, tries to read PWI4D.nii and quantifies with that in preference and only goes to PWI when PWI4D is not there. So all works as it should.
OK to resolve?

Reply:
OK I updated this in #997